src/main/java/wolox/training/models/User.java

```diff
@@ -17,7 +17,7 @@
     @Column(nullable = false)
     private LocalDate birthdate;
 
-    @OneToMany
+    @ManyToMany(cascade = CascadeType.ALL)
     @JoinColumn
```

Comment:
Won't work. ManyToMany implies a JoinTable, and it's not necesarry to specify it unless you want custom names for the table and/or it's columns.